app/views/shared/_landingheader.erb

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
This file seems to contain everything that also appears in _landingbodyheader.html.erb.  Are we duplicating the header stuff?  Is this perhaps dead code?

Reply:
Please take a closer look at landingbodyheader -- different content.   Here's the first 3 lines:
```<!-- Header -->
    <header>
        <div class="container">
            <a href="index.html" class="logo">```

Reply:
fixed.